### PR TITLE
Fix image timestamp displaying UTC time

### PR DESCRIPTION
**Pull request by bxtsang**

### CODE REVIEW

File: webproject/src/main/webapp/js/chat-box.js [hunk 1]
```diff
@@ -18,7 +18,12 @@ $(document).ready(function() {
         } else {
             html += "<a href=\"" + snap.message + "\"><img src=\"" + snap.message + "\" /></a> <br />";
         }
-        html += "<span class='chat-time'>" + snap.time + "</span>";
+
+        let date = new Date(snap.time);
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/chat-box.js]: rename snap to newMessage

File: webproject/src/main/java/com/google/sps/data/Message.java [hunk 2]
```diff
@@ -6,12 +6,12 @@
 
 public class Message {
     private String user;
-    private String time;
+    private long time;
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/data/Message.java]: rename to timestamp

File: webproject/src/main/webapp/js/chat-box.js [hunk 3]
```diff
@@ -18,7 +18,12 @@ $(document).ready(function() {
...
+        let minutes = minutes_with_leading_zeroes(date);
+        let hours = hours_with_leading_zeroes(date);
+
+        html += `<span class='chat-time'>${hours}:${minutes}</span>`;
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/chat-box.js]: should we add date here too?

File: webproject/src/main/webapp/js/chat-box.js [hunk 4]
```diff
@@ -40,7 +45,7 @@ function sendMessage() {
         var message = messageBox.value;
         messageBox.value = "";
         var date = new Date();
-        var time = hours_with_leading_zeroes(date) + ":" + minutes_with_leading_zeroes(date);
+        var time = date.getTime();
 
         firebase.database().ref('messages/' + roomID).push().set({
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/chat-box.js]: change this to send to servlet instead of firebase directly

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/webapp/js/chat-box.js (hunk 3 above)
Comment (bxtsang) [reply webproject/src/main/webapp/js/chat-box.js]: added!

File: webproject/src/main/java/com/google/sps/data/Message.java [hunk 5]
```diff
@@ -6,14 +6,14 @@
...
 public class Message {
     private String user;
-    private String time;
+    private long timestamp;
     private String message;
     private String type;
 
     public Message(String user, String imageUrl, String type) {
-        String currTime = new SimpleDateFormat("HH:mm").format(new Date());
+        long currTime = System.currentTimeMillis();
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/data/Message.java]: use timestamputil

**Review by kevinluvian — CHANGES_REQUESTED** [review]

**Review by kevinluvian — APPROVED** [review]

**Pull request merged**